presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,59 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Add spatial join (broadcast and distributed) support for :func:`ST_Equals`,
+  :func:`ST_Overlaps`, :func:`ST_Crosses`, and :func:`ST_Touches`.
+* Fix a bug where index joins would fail with the error "driver should never block".
+  Now queries will wait until a configured timeout for the indexes to load.  The
+  timeout can be configured by the configuration property ``index-loader-timeout``
+  and the session property ``index_loader_timeout``.
+* Add ``task_partitioned_writer_count`` session property to allow setting the number
+  of concurrent writers for partitioned (bucketed) writes.
+* Remove syntax support for ``SET PATH`` and ``CURRENT_PATH``. This information was
```

Comment:
Remove should go towards the end of the section. Updated [Order of changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes).